test/parser.mjs

```diff
@@ -7,16 +7,24 @@ const fixtures_dir = process.argv[2];
 
 if (!fixtures_dir) {
     console.error(
-        "Usage: node --experimental-modules parser.mjs FIXTURES_DIR");
+        "Usage: node --experimental-modules parser.mjs FIXTURES_DIR_OR_FTL");
     process.exit(1);
 }
 
 main(fixtures_dir);
 
 async function main(fixtures_dir) {
-    const files = await readdir(fixtures_dir);
-    const ftls = files.filter(
-        filename => filename.endsWith(".ftl"));
+    let files, ftls;
+    if (fixtures_dir.endsWith(".ftl")) {
+        // Actually, this is a filepath, split the path and the dir.
+        ftls = [path.basename(fixtures_dir)];
```

Comment:
This looks like a good place to use a `var`! You can declare `ftls` as `var` both here and inside of the `else` block.